packages/ui/src/components/Notification/index.tsx

```diff
@@ -0,0 +1,144 @@
+import type { Theme } from '@emotion/react'
+import { ClassNames, Global, css, useTheme } from '@emotion/react'
+import type { ReactNode } from 'react'
+import {
+  ToastContainer as BaseToastContainer,
+  Slide,
+  toast as baseToast,
+} from 'react-toastify'
+import type {
+  Theme as ThemeToastify,
+  ToastOptions,
+  TypeOptions,
+} from 'react-toastify'
+import style from 'react-toastify/dist/ReactToastify.min.css'
+import { Button } from '../Button'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+
+const PREFIX = '.Toastify'
+type StylesProps = {
+  theme: Theme
+}
+
+type CloseButtonProps = {
+  closeToast: (e: React.MouseEvent<HTMLElement>) => void
+  type: TypeOptions
+  ariaLabel?: string
+  theme: ThemeToastify
+}
+
+const styles = {
+  toast: ({ theme }: StylesProps) => css`
+    border-radius: ${theme.radii.default};
+
+    &${PREFIX}__toast {
+      background-color: ${theme.colors.neutral.background};
+      color: ${theme.colors.neutral.text};
+      padding: ${theme.space['2']};
+      box-shadow: ${theme.shadows.defaultShadow};
+    }
+
+    &${PREFIX}__toast-container {
+      width: 312px;
+    }
+
+    ${PREFIX}__toast-body {
+      margin: 0;
+      padding: 0;
+    }
+  `,
+}
+
+const closeButton = (props: CloseButtonProps) => (
+  <Button
+    aria-label="close"
+    icon="close"
+    sentiment="neutral"
+    variant="ghost"
+    onClick={props.closeToast}
+    size="xsmall"
+  />
+)
+
+export const notification = (
+  children: ((props: CloseButtonProps) => ReactNode) | ReactNode,
+  title: string,
+  icon?: ReactNode,
+  isClosable?: boolean,
+  options?: ToastOptions,
+) =>
+  baseToast('', {
+    ...options,
+    closeButton: props => (
+      <Stack direction="row" gap={2}>
+        <div> {icon}</div>
+        <Stack direction="column">
+          <Text as="div" variant="bodySmallStronger">
```

Comment:
```suggestion
          <Text as="h3" variant="bodySmallStronger">
```

In this case we have a title so it's better to use the correct tag